test/Libraries/Microsoft.Extensions.AI.Evaluation.NLP.Tests/BLEUAlgorithmicTests.cs

```diff
@@ -0,0 +1,246 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using Microsoft.Extensions.AI.Evaluation.NLP.Common;
+using Xunit;
+using static Microsoft.Extensions.AI.Evaluation.NLP.Common.BLEUAlgorithm;
+
+namespace Microsoft.Extensions.AI.Evaluation.NLP.Tests;
+
+[Experimental("AIEVAL001")]
+public class BLEUAlgorithmicTests
```

Comment:
minor: I don't feel too strongly but we have generally used the `<ClassName>Tests` naming convention

```suggestion
public class BLEUAlgorithmTests
```